packages/messaging/e2e/messaging.e2e.js

```diff
@@ -36,9 +36,12 @@ describe('messaging()', () => {
     });
 
     it('sets the value', async () => {
-      should.equal(firebase.messaging().isAutoInitEnabled, true);
-      await firebase.messaging().setAutoInitEnabled(false);
-      should.equal(firebase.messaging().isAutoInitEnabled, false);
+      // FIXME failing in CI
```

Comment:
this one works for me as well but it did start failing locally. I investigated and I think it's the `test-reuse` case of local e2e execution since we weren't resetting the state at the end. In my separate string of fixes I started resetting to default value and asserting it was true again, and it passes.